pkg/kubelet/container/runtime.go

```diff
@@ -89,6 +89,27 @@ type Container struct {
 	Created int64
 }
 
+// RunContainerOptions specify the options which are necessary for running containers
+type RunContainerOptions struct {
+	// The environment variables, they are in the form of 'key=value'.
+	Envs []string
+	// The mounts for the containers, they are in the form of:
+	// 'hostPath:containerPath', or
+	// 'hostPath:containerPath:ro', if the path read only.
+	Binds []string
+	// If the container has specified the TerminationMessagePath, then
+	// this path will be used to mount to that path.
+	ContainerLogPath string
+	// The DNS settings for the container.
+	DNS       []string
+	DNSSearch []string
+	// Namespaces for the container.
```

Comment:
Ah this is for the runtime, we'll need more description here since its not clear. Which does Rocket support